cmd/get/catalogs/runner.go

```diff
@@ -107,7 +107,15 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 		}
 	}
 
-	err = r.printOutput(catalogResource)
+	var maxColWidth uint
+	{
+		maxColWidth, err = cmd.InheritedFlags().GetUint("max-col-width")
+		if err != nil {
+			return microerror.Mask(err)
+		}
```

Comment:
@rossf7 is just that I wanted it to be sort of global to all the get sub-commands without definining it in each of them

Reply:
Do we need it in the other get sub-commands?

You could use the same approach we use to add the upstream print flags. Otherwise not sure how to do this.

https://github.com/giantswarm/kubectl-gs/blob/5e7577ad47e575c7557208800a3b6d81da895c60/cmd/get/catalogs/flag.go#L25-L28

Reply:
> Do we need it in the other get sub-commands?
Well, not sure, sounds like it should be common to all the command that prints something, but anyway, I don't see any problem moving it downstream to the sub-command that currently uses it. Anyway it just a cosmetic change and we can figure out how to make it common when we will actually need it.

Reply:
Cool this looks good and if we need this in other commands we can adjust later.